samples/bluetooth/ipsp/README.rst

```diff
@@ -22,6 +22,16 @@ Sample can be built and executed for the nRF52840 DK NRF52840 as follows:
    :goals: build flash
```

Comment:
For commit `samples: ipsp: add a debug config build example`, you need to add a commit description (it cannot be left empty). I wonder why checkpatch did not complain about this.

Reply:
Oh my bad, didn't rerun it. There was also a multiline comment warning in the first patch, fixed that one too.